Kinvey/SSOApp/SSOApp1/AppDelegate.swift

```diff
@@ -51,7 +51,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     }
     
     func application(_ application: UIApplication, open url: URL, options: [UIApplicationOpenURLOptionsKey : Any] = [:]) -> Bool {
-        if User.login(redirectURI: micRedirectURI, micURL: url, authServiceId: authServiceId) {
+        if User.login(redirectURI: micRedirectURI, micURL: url, options: Options(authServiceId: authServiceId)) {
```

Comment:
Is this a sample app? Did a new warning appear due to an API change made sometime earlier to the SDK?

Reply:
yeah, same situation